tests/specs/api/menu.spec.js

```diff
@@ -0,0 +1,107 @@
+import * as httpMock from '../../mocks/http';
+import { getAuthenticationHeaders } from '../../../src/utils/auth';
+import { MenuApi } from '../../../src/api/menu';
+
+
+describe('Menu API', () => {
+    const serviceUrl = 'http://some-url.com';
+    const invalidAuthErrorMessage = 'Must not use an app token for authentication.';
+    const noPropsMessage = 'Must provide props.';
+    const noItemsMessage = 'Must provide an array of items.';
```

Comment:
I am not sure if we should validate with message strings? Can you match them up with error/return codes?